cores/nRF5/Tone.cpp

```diff
@@ -332,14 +332,8 @@ bool TonePwmConfig::applyConfiguration(uint32_t pin) {
     nrf_pwm_shorts_set(_PWMInstance, this->shorts);
     nrf_pwm_int_set(_PWMInstance, 0);
 
-    // static sequence value ... This is the value actually used
-    // during playback ... do NOT modify without semaphore *and*
-    // having ensured playback has stopped!
-    static uint16_t seq_values; // static value 
```

Comment:
Do you realize that, by this change, you must add synchronization to `this->duty_with_polarity`? (or otherwise ensure not playing before changing this).

This makes the code more _fragile_; I recommend against this change.

Reply:
I don't see any issue with seq got changed. it only happens when use call tone() again, it is OK to got value changed.

Reply:
If you cannot see the problem, I give up.

Reply:
AFAIK initializeFromPulseCountAndTimePeriod() is the only function that change the `duty_with_polarity`, And the playback is stopped before it is called. There should be no conflict when value changed when playing back, or I am missing something  !!